spec/requests/partners/profiles_requests_spec.rb

```diff
@@ -32,5 +32,25 @@
       expect(partner.profile.address2).to eq("Washington, DC")
       expect(response).to redirect_to(partners_profile_path)
     end
+
+    context "when updating an existing value to a blank value" do
+      before do
+        partner.profile.update!(city: "")
+        put partners_profile_path(partner,
+          partner: {name: "Partnerdude", profile: {city: "", website: "N/A"}})
+      end
+
+      it "updates the partner profile attribute to a blank value" do
+        expect(partner.profile.city).to eq ""
+      end
+
+      it "does not update other partner profile attributes to blank" do
+        expect(partner.profile.address2).to be_nil
+      end
+
+      it "does not store N/A in the database" do
+        expect(partner.profile.website).to be_nil
```

Comment:
Wait, is there something that deliberately changes the value "N/A" to nil? If this test passes, I'm not sure why.

Reply:
Not really, It was because I forgot to add reload, it should be `partner.profile.reload.website` as I a use `put` request in `before` block. The caching is causing the problem. I will update with the changes.